app/src/main/java/com/framgia/wsm/utils/Constant.java

```diff
@@ -39,4 +39,19 @@ private Constant() {
         public static final String LEAVE_EARLY_WOMAN_M = "Leave early (woman only) (M)";
         public static final String LEAVE_EARLY_WOMAN_A = "Leave early (woman only) (A)";
     }
+
+    public static class Time {
+        public static int SEVEN_HOUR = 7;
+        public static int EIGHT_HOUR = 8;
+        public static int NIGHT_HOUR = 9;
+        public static int TEN_HOUR = 10;
+        public static int ELEVEN_HOUR = 11;
+        public static int TWELVE_HOUR = 12;
+        public static int THIRTEEN_HOUR = 13;
+        public static int FOURTEEN_HOUR = 14;
```

Comment:
ERROR: [MALICIOUS_CODE][MS_SHOULD_BE_FINAL] com.framgia.wsm.utils.Constant$Time.FOURTEEN_HOUR isn't final but should be
http://findbugs.sourceforge.net/bugDescriptions.html#MS_SHOULD_BE_FINAL